main.tf

```diff
@@ -243,9 +240,7 @@ resource "aws_autoscaling_group" "bastion_auto_scaling_group" {
   min_size             = "${var.bastion_instance_count}"
   desired_capacity     = "${var.bastion_instance_count}"
 
-  vpc_zone_identifier = [
-    "${var.auto_scaling_group_subnets}",
-  ]
+  vpc_zone_identifier = "${var.auto_scaling_group_subnets}"
```

Comment:
Nitpick: Variable references. I see some variables you switched to or added are using the TF 0.12 syntax, a handful like this are still using the interpolation syntax when just being referenced.

Reply:
Fixed